ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/materialization/ks/KsMaterializedTableIQv2Test.java

```diff
@@ -127,7 +127,6 @@ public void shouldThrowIfQueryFails() {
     );
 
     // Then:
-    System.out.println(e.getMessage());
```

Comment:
This was leftover from test debugging. I didn't want to hold up the other PR on this one fix, so I'm doing it after the merge.